.github/workflows/build-and-test-other.yaml

```diff
@@ -22,17 +22,22 @@ jobs:
 
```

Comment:
This PR has been merged, I think we can rebase on master.

Reply:
I guess we only needed to restart merge, the first commit was the one from #398